Fix link from ARA handbook (#6803)
GoogleChrome/developer.chrome.com: site/en/docs/privacy-sandbox/attribution-reporting-debugging/part-2/index.md

{% endDetailsSummary %}

> - Check that your code is gated behind feature detection. Before you use the API, detect if it hasn't been disallowed on the page via a Permissions-Policy. To do so, run the following code:

Reviewer: "Before you use the API, check if it hasn't been blocked from the page because of a Permissions-Policy."

This one's a suggestion that I think is easier to understand—the via should definitely go, it makes it easier to misread, the "blocked" i'm less sure about. Might be worth checking with Rachel, perhaps?

  ```javascript
  if (document.featurePolicy.allowsFeature('attribution-reporting')) {
  // the Attribution Reporting API is enabled
  }
  ```

  If this feature detection check returns true, the API is allowed in the context (page) where the check is run. 
- (Not required during the testing phase: Check that you've set a [Permissions-Policy](https://docs.google.com/document/d/1BXchEk-UMgcr2fpjfXrQ3D8VhTR-COGYS1cwK_nyLfg/edit#heading=h.ju0kr1sopvhz))

{% Aside 'gotchas' %}

> Permissions-Policy requirements have been [relaxed](https://docs.google.com/document/d/1BXchEk-UMgcr2fpjfXrQ3D8VhTR-COGYS1cwK_nyLfg/edit#bookmark=id.qoybi4lx0dka) for testing, but a publisher or advertiser can still decide to explicitly disable the Attribution Reporting API via Permissions-Policy. In this case, no source nor trigger can be recorded (and hence no report can be generated or sent).

Reviewer: ```suggestion
Permissions-Policy requirements have been [relaxed](https://docs.google.com/document/d/1BXchEk-UMgcr2fpjfXrQ3D8VhTR-COGYS1cwK_nyLfg/edit#bookmark=id.qoybi4lx0dka) for testing, but a publisher or advertiser can still decide to explicitly disable the Attribution Reporting API using the Permissions-Policy. In this case, no source or trigger can be recorded (and therefore, no report can be generated or sent).
```

{% endDetailsSummary %}

> - Check that your code is gated behind feature detection. To be sure the API is not blocked by Permissions-Policy, run the following code:

Reviewer: ```suggestion
- Check that your code is gated behind feature detection. To ensure that the API isn't blocked on the page by Permissions-Policy, run the following code:
```
